ecs-init/cache/cache.go

```diff
@@ -76,11 +91,34 @@ func NewDownloader() (*Downloader, error) {
 	return downloader, nil
 }
 
+func (d *Downloader) AgentCacheStatus() CacheStatus {
+	stateFile := config.CacheState()
+	cacheControlled := d.fileNotEmpty(stateFile) && d.fileNotEmpty(config.AgentTarball())
```

Comment:
can this be named just `uncached`?

Reply:
This is more like `cached`, I called it `cacheControlled` in that the presence of both non-empty files makes this a reportable status.